vsdehalo/denoise.py

```diff
@@ -70,15 +73,21 @@ def bidehalo(
 
     planes = normalise_planes(clip, planes)
 
+    if isinstance(bm3d_sigma, (int, float)):
+        bm3d_sigma = [float(bm3d_sigma)] * 2
```

Comment:
Consider scaling the value for chroma instead of using the same sigma for luma and chroma